scripts/install_core_submodules.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
```

Comment:
Don't copy-paste this scripts. Divide scripts on small ones and use their if you needed